tests/unit/lms/views/helpers/frontend_app_test.py

```diff
@@ -35,89 +28,40 @@ def test_it_disables_grading_if_grading_is_disabled_for_assignment(
 
         assert "lmsGrader" not in js_config
 
-    def test_it_fetches_grading_info_records(
-        self, grading_request, grading_info_service
-    ):
-        frontend_app.configure_grading(grading_request, {})
+    def test_it_enables_grading(self, grading_request, grading_info_service):
+        js_config = {}
+
+        frontend_app.configure_grading(grading_request, js_config)
 
+        assert js_config["lmsGrader"] is True
         grading_info_service.get_by_assignment.assert_called_once_with(
             oauth_consumer_key=grading_request.lti_user.oauth_consumer_key,
             context_id=grading_request.params["context_id"],
             resource_link_id=grading_request.params["resource_link_id"],
         )
-
-    def test_it_sets_list_of_grading_info_on_config(self, grading_request):
-        js_config = {}
-
-        frontend_app.configure_grading(grading_request, js_config)
-
-        assert "students" in js_config["grading"]
-        assert js_config["grading"]["students"] == []
-
-    def test_it_sets_js_properties_for_each_grading_info_record(
-        self, grading_info_service, grading_infos, grading_request
-    ):
-        js_config = {}
-        grading_info_service.get_by_assignment.return_value = grading_infos
-
-        frontend_app.configure_grading(grading_request, js_config)
-
-        assert len(js_config["grading"]["students"]) == 3
-        for propName in [
-            "userid",
-            "displayName",
-            "LISResultSourcedId",
-            "LISOutcomeServiceUrl",
-        ]:
-            assert propName in js_config["grading"]["students"][0]
-
-    def test_it_sets_course_and_assignment_names(self, grading_request):
-        js_config = {}
-        frontend_app.configure_grading(grading_request, js_config)
-
-        assert js_config["grading"]["courseName"] == "Test Course 101"
-        assert js_config["grading"]["assignmentName"] == "How to use Hypothesis"
-
-
-pytestmark = pytest.mark.usefixtures("grading_info_service")
-
-
-@pytest.fixture
-def grading_infos():
-    grading_infos = []
-
-    for index in range(3):
-        h_username = f"username_{index}"
-        h_display_name = f"Student {index}"
-        lis_result_sourcedid = f"lis_result_sourcedid_{index}"
-
-        grading_infos.append(
-            (
-                GradingInfo(
-                    h_username=h_username,
-                    h_display_name=h_display_name,
-                    lis_result_sourcedid=lis_result_sourcedid,
-                    lis_outcome_service_url="http://example.com/service_url",
-                ),
-                HUser("test_authority", h_username, h_display_name),
-            )
```

Comment:
Massive improvement here. The tests were three times as long as the code (that's not rare, but it's nicer if it's not true)